client/tos/index.js

```diff
@@ -0,0 +1,39 @@
+/* global wcpay_tos_settings */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import TosModal from './modal';
+import TosSnackbar from './snackbar';
+
+// eslint-disable-next-line camelcase
+if ( wcpay_tos_settings.showModal ) {
```

Comment:
minor thing but `showModal` reads like an action rather than boolean flag. `tosAgreementRequired` as in php sounds better IMO.

Reply:
It is called `tosAgreementRequired` now: https://github.com/Automattic/woocommerce-payments/pull/993/commits/8ad28b977f3e8bb150397473aadadaf845edfb29#diff-3863b4f88d0515e080fadc56af3c5e38e942afb96e569cb000ae6a594e8bea4aR314